frontend/src/components/LoginForm.tsx

```diff
@@ -60,9 +62,33 @@ export default function LoginForm({
   const history = useHistory()
 
   const { from } = location.state || { from: { pathname: "/" } }
+  const [isOpen, setOpen] = React.useState<boolean>(false)
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [type, setType] = useState<NotificationType["type"]>("error")
+  // eslint-disable-next-line @typescript-eslint/no-unused-vars
+  const [message, setMessage] = useState(
+    "Email and/or password entered incorrectly"
+  )
```

Comment:
why not set the type and message when you catch the error?

Reply:
I could.. Lowkey wouldn't make much difference but I'll have that done.

Reply:
Instead of the eslint disable comments, just replace `const [foo, setFoo] = ...` with `const [foo] = ...`